src/xorIt.js

```diff
@@ -10,23 +6,19 @@
  * http://cwestblog.com/2013/08/19/javascript-simple-vernam-cipher/
  */
 
-var xorIt = (key, textOrCipher) => {
+export default function xorIt(key, textOrCipher) {
   if (!key || !textOrCipher) {
     throw 'XorIt: needs key and textOrCipher';
   }
 
-  let cipher = [],
-      keyLength = key.length,
-      fromCharCode = String.fromCharCode,
-      len = textOrCipher.length,
-      i = 0;
+  let cipher = [];
```

Comment:
nvm
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const
The const declaration creates a read-only reference to a value. It does not mean the value it holds is immutable, just that the variable identifier cannot be reassigned.